src/asciidoc/docs/customizations/media.adoc

```diff
@@ -0,0 +1,220 @@
+= Media
+:page-slug: media
+:page-description: Standard VCC for selecting an image, file, sound, or video.
+:figure-caption!:
+
+Users can
+//tag::description[]
+select an image, file, sound, or video by selecting from the available options, uploading, or entering a URL.
+//end::description[]
+
+image::mediaVCC.png[title="Media VCC"]
+
+== Sample media VCC
+
+[source,json]
+----
+{
+    "key": "imageOrVideo",
+    "name": "Image or Video",
+    "type": "media",
+    "typeOptions": {
+        "acceptOnly": ["image", "video"],
+        "videoOptions": {
+            "hls": true
+        }
+    }
+}
+----
+
+== Type options
+
+The following code sample shows all the available `typeOptions`.
+Each option is described in the sections that follow the sample.
+
+[source,json]
+----
+{
+    "name": "Media Upload",
+    "type": "media",
+    "typeOptions": {
+        "acceptOnly": ["image", "video", "audio"],
+        "returnType": "url"|"extended",
+        "hideExtensions": false,
+        "videoOptions": {
+            "hls": true,
+            "estimatePoses": false
+        },
+        "imageOptions": {
+            "width": 100,
+        },
+        "audioOptions": {},
+        "fileOptions": {}
+    }
+}
+----
+
+=== acceptOnly
+
+`acceptOnly` allows you to specify the VCC types that are allowed.
+The value for `acceptOnly` is an array of one or more VCC types.
+If the array contains more than one type, then the user can choose any file of a type that matches one of the types in the array.
+For example, `"acceptOnly": ["image", "video"]` allows the user to select an image file or a video file.
+If the value of `acceptOnly` is an empty array, or if `acceptOnly` is not included in `typeOptions`, then any file type will be allowed.
+
+The possible types are as follows.
+
+* *image* - If you set `acceptOnly` to an array containing `image`, the user will be prompted to choose an image file for the VCC.
```

Comment:
These descriptions contain lots of redundant text that doesn't add clarity. Try simplifying, e.g.,

"* *image* – Allows users to choose images files.
Setting `acceptOnly` to `["image"]` is equivalent to using an <<image#, image VCC>>."